### DISCUSSION TITLE
Comments disable on posts with "remark42" chapter

### DISCUSSION THREAD

**Discussion by halon176** [umputun/remark42#1173] [General]
i think i have found a bug.

On my blog I created a post where I told the passage to remark42, when I went to generate the static site the comments on that article did not work. After spending a lot of time doing the surveys, I realized that if I leave the title of a chapter "### remark42" the page where it is present does not generate the comment form.

But adding anything in front eg. "### gg remark42" or comments are enabled again.

I hope it will be useful to someone.

**Comment by umputun** [comment]
thx for the report, but I'm having a hard time parsing it. It sounds like your remark42 integrated into some system with some third-party module in some way. I have very little clue what it means and details will help to realize what this is about. All I can see is some chapters (?) somewhere and somehow linked to remark42 and this information is not sufficient to figure what the problem is.

**Reply by halon176** [reply]
I agree with you that it is a minor problem, I wanted to report it because I was going crazy to understand why the comment form was not generated in that post.

In any case, I remain available in case you need more information to solve it.

**Reply by umputun** [reply]
I'm not saying it is minor, just trying to understand your setup and figure what "I leave the title of a chapter "### remark42" the page where it is present does not generate the comment form" actually means

**Reply by halon176** [reply]
ah ok, by bad.

i'm using hugo v0.89.2+extended, with hugo-PaperMod
remark works perfectly on every page

if i try to insert in a post any kind of heading like

`### remark42`
or
`# remark42`

the post is generated without comment form

I hope it is clearer now

**Reply by umputun** [reply]
yeah, it helps. But how exactly is it integrated? Is this a part of PaperMod or you did it by yourself?

**Reply by halon176** [reply]
> yeah, it helps. But how exactly is it integrated? Is this a part of PaperMod or you did it by yourself?

no no it is just a post i write in /content/posts, i'm using fresh versione of papermod and I haven't touched anything from the default settings

if you want to see the integration I made with remark [you can find it here](https://github.com/halon176/halon/blob/main/layouts/partials/comments.html) but it's nothing special

**Reply by umputun** [reply]
ok, got it now, thx for providing the info. I would guess on a page with the missing remark42 widget you would see some errors in the console. I suspect something around permalinks but if you can check what the error exactly is - this may help

cc: @akellbl4
